addons/website_quote/__openerp__.py

```diff
@@ -11,11 +13,11 @@
         """,
     'depends': ['website', 'sale', 'mail', 'payment', 'website_portal_sale', 'website_mail'],
     'data': [
-        'quotation_report.xml',
+        'report/quotation_report.xml',
         'views/website_quotation.xml',
-        'views/website_quotation_backend.xml',
+        'views/website_quotation_views.xml',
         'views/report_saleorder.xml',
```

Comment:
Okay, I thought this file should remain there as the similar file in the already migrated module website_sale wasn't moved ("report_shop_saleorder.xml").

Shall I move report_shop_saleorder.xml to a report folder in website_sale as well ? Or is it bad practice to do that in this branch ?